br/pkg/backup/client.go

```diff
@@ -95,7 +95,7 @@ type Client struct {
 
 	cipher           *backuppb.CipherInfo
 	checkpointMeta   *checkpoint.CheckpointMetadata
-	checkpointRunner *checkpoint.CheckpointRunner
+	checkpointRunner *checkpoint.CheckpointBackupRunner
```

Comment:
when referencing `CheckpointBackupRunner` outside the package, it will be `checkpoint.checkpointBackupRunner`, which is a little redundancy. We can rename it to `BackupRunner`